bcipy/acquisition/README.md

```diff
@@ -1,12 +1,66 @@
 # Data Acquisition Module (daq)
 
-The Data Acquisition module is intended to be used within the Brain Computer Interface (BCI) systems. This module streams data from the EEG hardware, persists it in csv files, and makes it available to other systems.
+The data acquisition module is responsible for interfacing with hardware to obtain device data (EEG, eye tracking, etc). It supports multi-modal recording, real-time querying, and functionality to mock device data for testing.
+
+## Supported Devices
+
+The acquisition module connects with hardware devices using the Lab Streaming Layer (LSL) library. Each device should provide its own LSL driver / application to use for streaming data. The streamer should be started prior to running the `bcipy` application.
```

Comment:
Examples could be helpful here and/or a link to an app! 

Apps: https://labstreaminglayer.readthedocs.io/info/supported_devices.html
If your device does not have a supported app see https://labstreaminglayer.readthedocs.io/dev/app_build.html